packages/cf-builder-pagination/test/PaginationBuilder.js

```diff
@@ -86,7 +85,7 @@ test('should should call onPageChange when clicking another page', () => {
     />
   );
 
-  const items = wrapper.find(PaginationItem);
+  const items = wrapper.find('li');
```

Comment:
Explained above. Not sure why Enzyme doesn't like this anymore but the is still legit (testing the same functionality).